frontend/database/106_remove_duplicated_schools.sql

```diff
@@ -0,0 +1,90 @@
+# Take just one of the duplicated elements as the one that will remain id DB.
+# Update the relations with Courses
+UPDATE
+    `Courses`
+INNER JOIN (
+    SELECT
+        `sc`.`school_id`, `min_id`
+    FROM
+        `Schools` `sc`
+    INNER JOIN (
+        SELECT
+            `name`, min(`school_id`) as `min_id`
+        FROM
+            `Schools`
+        GROUP BY
+            `name`
+        HAVING
+            COUNT(*) > 1
+    ) `dups` # duplicates
```

Comment:
```suggestion
    ) AS duplicates
```

qué te parece si en vez de tener que agregar un comentario, mejor usas ese nombre para la variable? así es más legible!

igual en los demás lugares donde se hace esto mismo.